recipes/ede-php-autoload-composer-installers

```diff
@@ -0,0 +1,3 @@
+(ede-php-autoload-composer-installers :fetcher github
+                                      :repo "xendk/ede-php-autoload-composer-installers"
+                                      :files (:defaults "ede-php-autoload-composer-installers"))
```

Comment:
There is no `ede-php-autoload-composer-installers` file/directory in repository.